Explorer/Assets/Scripts/ECS/SceneLifeCycle/Systems/DestroySceneSystem.cs

```diff
@@ -0,0 +1,42 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using ECS.Abstract;
+using ECS.SceneLifeCycle.Components;
+
+namespace ECS.SceneLifeCycle.Systems
+{
+    [UpdateInGroup(typeof(SimulationSystemGroup))]
+    [UpdateAfter(typeof(SceneLoadingSystem))]
+    public partial class DestroySceneSystem : BaseUnityLoopSystem
+    {
+        public DestroySceneSystem(World world) : base(world) { }
+
+        protected override void Update(float t)
+        {
+            DestroyLiveSceneQuery(World);
+            DestroyLoadingSceneQuery(World);
+        }
+
+        [Query]
+        [All(typeof(DeleteSceneIntention), typeof(LiveSceneComponent))]
+        private void DestroyLiveScene(in Entity entity)
+        {
+            var liveScene = World.Get<LiveSceneComponent>(entity);
```

Comment:
just move it to the arguments list, then you won't need to indicate `[All(...typeof(LiveSceneComponent))]` explicitly